### PR TITLE
added css & js files

### PR DESCRIPTION
**Pull request by cmdge**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="css/style.css">
+  <title>Do it!</title>
```
Comment (laureanray) [inline index.html:7]: Please make the commit smaller next time.

**Review by laureanray — APPROVED** [review]
Make sure the commit message is concise and understandable.

File: index.html (hunk 1 above)
Comment (laureanray) [reply index.html:7]: Also, you forgot to add the bootstrap.css

Comment (cmdge) [reply index.html:7]: Noted and thanks, LR!

**Pull request merged**